### PR TITLE
WiP: Dev mariateo skills

### PR DESCRIPTION
**Pull request by MariaTeo**
Started Skills section 
Styled the legend and the list
Also I noticed a mistake

### CODE REVIEW

File: src/_component/about-me/about-me.scss [hunk 1]
```diff
@@ -66,7 +66,7 @@ button[class^="pill"] i[class^="icon__"]{
   line-height: 48px;
 }
 
-.about-me__more-tile{
+.about-me__more-title{
```
Comment (RalucaRbc) [inline src/_component/about-me/about-me.scss:69]: Think you need an empty space between the div and the `{`

**Review by RalucaRbc — COMMENTED** [review]
Looks fine so far, gj Maria 👍

**Comment by MariaTeo** [comment]
> Looks fine so far, gj Maria 👍 

Thank you ☺️

**Review by QuintyHH — APPROVED** [review]
💯

**Review by biggicode — APPROVED** [review]
👍

**Comment by NicolaePaulGeambazu** [comment]
(Y)

**Review by tandaraelena — APPROVED** [review]

**Review by psyhedeliq — APPROVED** [review]
Nice work! 🚀

**Pull request merged**